platforms/fabric/src/main/resources/fabric.mod.json

```diff
@@ -32,9 +32,9 @@
         "terra.common.mixins.json"
     ],
     "depends": {
-        "fabricloader": ">=0.16.5",
+        "fabricloader": ">=0.16.7",
         "java": ">=21",
-        "minecraft": ">=1.21.1",
+        "minecraft": ">=1.21.2-beta.2",
```

Comment:
the `-beta.2` can probably be dropped here

Reply:
about to do that